src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -277,4 +279,87 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
       ),
     );
   });
+
+  it('should notify user that the browser is blocked from using current location information', async () => {
+    const store = createTestStore(initialState);
+
+    mockGetCurrentPosition({ fail: true });
+
+    mockCCProviderFetch(
+      initialState.user.profile.vapContactInfo.residentialAddress,
+      ['208D00000X', '207R00000X', '261QP2300X'],
+      calculateBoundingBox(
+        initialState.user.profile.vapContactInfo.residentialAddress.latitude,
+        initialState.user.profile.vapContactInfo.residentialAddress.longitude,
+        60,
+      ),
+      CC_PROVIDERS_DATA,
+    );
+
+    await setTypeOfCare(store, /primary care/i);
+    await setTypeOfFacility(store, /Community Care/i);
+
+    const screen = renderWithStoreAndRouter(
+      <CommunityCareProviderSelectionPage />,
+      {
+        store,
+      },
+    );
+
+    // Choose Provider
+    userEvent.click(await screen.findByText(/Choose a provider/i));
+    userEvent.click(await screen.findByText(/use your current location/i));
+
+    expect(
+      await screen.findByText(
+        /Your browser is blocked from finding your current location. Make sure your browser’s location feature is turned on./i,
+      ),
+    ).to.be.ok;
+  });
+
+  it('should sort by distance from current location (ascending) if user clicks "use current location"', async () => {
```

Comment:
It doesn't look like we choose current location in here

Reply:
Done.